app/src/main/java/com/kickstarter/libs/FirebaseHelper.kt

```diff
@@ -27,21 +33,22 @@ class FirebaseHelper(context: Context, ffClient: FeatureFlagClientType, callback
     }
 
     init {
-        if (context.isKSApplication()) {
-            if (FirebaseApp.getApps(context).isEmpty()) {
-                FirebaseApp.initializeApp(context)
-                FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(true)
-                FirebaseAnalytics.getInstance(context).setAnalyticsCollectionEnabled(true)
-            }
+        // not sure if runBlocking is the correct coroutine launcher we want here, but it seems to work on normal app bootstrap
+        runBlocking {
```

Comment:
@leighdouglas what happens if you try to start the app without internet? 
Fwi: if you want an alternative to `runBlocking` you can use from `KSAplication` application scope, you just need to send it to FirebaseHelper.initialize
<img width="978" height="149" alt="Screenshot 2025-09-03 at 12 47 45 PM" src="https://github.com/user-attachments/assets/ef2d6ec4-f577-40c0-a6e2-d288d7704525" />

<img width="765" height="42" alt="Screenshot 2025-09-03 at 12 48 31 PM" src="https://github.com/user-attachments/assets/9be9e081-02a2-4ae9-bddc-785330243c0b" />

Reply:
I've reverted some of these changes to FirebaseHelper. 
I will have a better idea of what happens when the app starts without internet once I make the deeplink activity the launching activity. The plan is to emulate the behavior that we have right now and bring them into the app anyways. We can address no internet connection in the future